index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello World</h1>
+</body>
+</html>
```

Comment:
After setup linters correctly end of pull request you will see this. Make sure you fix errors and see green mark.
![image](https://user-images.githubusercontent.com/38635158/124671827-c11d2f80-debe-11eb-9801-54fb51b37c93.png)

Reply:
Hi @ahmetbozaci so actually what I am supposed to do is add linters properly and a readme file template to be followed. I just wanted to ask what do you means by comment on lines 11 and 13

Reply:
Hello @faizi2500.
![image](https://user-images.githubusercontent.com/38635158/124713638-7aa0f280-df09-11eb-9e62-a31735530639.png)
End of the html and css file you have linter errors. You should fix this. You will only add new line. Then error will fix. But firstly you should set up your linter files correctly. Original file:
![image](https://user-images.githubusercontent.com/38635158/124713918-cf446d80-df09-11eb-8da1-ec20f3526cf0.png)
You miss` hintrc and stylelintrc` files.

Reply:
I am lost actually, I dont know how to fix all of this

Reply:
So I tried the entire project on a new repo, it worked fine there. Is it ok if I submit the link to that repo in my dashboard. Because I am already behind in the curriculum, Ill try fixing this repo on the weekend.

Reply:
@faizi2500  You didn't setup linter files. Please check this [lesson](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up linter files correctly.  After finish requirements that I mention send review again in your dashboard..

Reply:
@faizi2500 My mistake. Your linter files setup correctly. But you have conflict in your branch so linter is not working because of that.
![image](https://user-images.githubusercontent.com/38635158/124758050-e2216700-df36-11eb-98d6-d8b8426659b1.png)
Please fix this then your linter files will work correctly. 
![image](https://user-images.githubusercontent.com/38635158/124759222-16e1ee00-df38-11eb-8d03-2147fafb8f8f.png)
For more information  check this [video](https://www.youtube.com/watch?v=JtIX3HJKwfo)

Reply:
Working on it. :)

Reply:
Is it possible to upload a new repository for review? I am totally lost with this one. 
I have spent alot of time on this one. Can't spend more. I am unable to click on resolve conflicts button

Reply:
Well resolved.